docs/Maintainer-Guidelines.md

```diff
@@ -105,6 +107,29 @@ is a good opportunity to do it) provided the line itself has some kind
 of modification that is not whitespace in it. But be careful about
 making changes to inline patches—make sure they still apply.
 
+### Adding formulae
+Only one maintainer is necessary to approve and merge the addition of a new formula which passes CI. However, if the formula addition
+is controversial the maintainer who adds it will be expected to fix issues that arise with it in future.
+
+### Removing formulae
+Formulae that:
+
+- work on at least 2/3 of our supported macOS versions in the default Homebrew prefix
+- do not require patches rejected by upstream to work
+- do not have known security vulnerabilities/CVEs for the version we package
+- are shown to be still installed by users in our analytics with a `BuildError` rate of <25%
+
+
+should not be removed from Homebrew. The exception to this rule are [versioned formulae](Versions.md) for which there are higher standards of usage and a maximum number of versions for a given formula.
+
+### Closing issues/PRs
+Maintainers (including the lead maintainer) should not close issues or pull requests opened by other maintainers unless they are stale (i.e. have seen no updates for 28 days) in which case they can be closed by any maintainer. If the close is undesirable: no big deal, they can be reopened when additional work will be done.
+
+Any maintainer can merge any PR passing CI that has been opened by any other maintainer. If you do not wish to have other maintainers merge your PRs: please use the `do not merge` label to indicate that until you're ready to merge it yourself.
+
+## Reverting PRs
+Any maintainer can revert a PR created by another maintainer after a user submitted issue or CI failure that results. The maintainer who created the original PR should be given an hour to fix the issue themselves or decide to revert the PR themselves if they would rather.
```

Comment:
An hour seems arbitrarily low, considering the maintainer may be busy (or not, being in the movie theatre would suffice) or asleep due to a different time zone. Depending on the severity of the issue, one hour may not be enough.

How about “should be given **no less than an hour** to fix the issue” (without the emphasis)? This way, it’s up to the reverter to judge the severity and make a call if it’s pressing or not.

Reply:
@vitorgalvao That sounds good 👍